### PR TITLE
Updating actions to clone Guides and run GuideConverter locally

### PR DESCRIPTION
**Pull request by jakub-pomykala**
The GuideConverter now runs on local guides, this means that the action has been changed to clone the repo of the guide (& guides-common) and then delete the these repos after. This is so that there is no downtime like there was when the GuideConverter was running using the GitHub API. In this PR there is also a change included to the `mirror.yml` files where I previously didn't add `cloud-hosted-` at the beginning of one of the files.

### CODE REVIEW

File: .github/workflows/DraftGuideAction.yml [hunk 1]
```diff
@@ -28,14 +28,26 @@ jobs:
   
     steps:
 
-      # Any prerequisite steps
+           # Any prerequisite steps
```
Comment (austin0) [inline .github/workflows/DraftGuideAction.yml:31]: Weird added tab.

**Review by austin0 — CHANGES_REQUESTED** [review]
Please update the PR title/description to reflect changes (as this is a pretty big change).

File: .github/workflows/DraftGuideAction.yml (hunk 1 above)
Comment (jakub-pomykala) [reply .github/workflows/DraftGuideAction.yml]: Done

**Review by austin0 — APPROVED** [review]
LGTM, passed a trial run on master.

**Pull request merged**